src/random.jl

```diff
@@ -100,3 +100,17 @@ function nichemodel(S::Int64, C::Float64; tolerance::Float64=0.05, toltype::Symb
     end
     return A
 end
+
+"""Compute number of species in the system from the trophic matrix."""
```

Comment:
Reserve use of triple quotes `"""` for multiline strings. Here a plain `"` will be fine ;)